hieradata/common.yaml

```diff
@@ -221,11 +221,11 @@ profile::buildmaster::agents_setup:
     osDiskSize: 90
 profile::buildmaster::agent_images:
   ec2_amis:
-    ubuntu-amd64: "ami-084ae16bc6401a89f"
-    windows-amd64: "ami-0dd4af40c365047fa"
-    ubuntu-arm64: "ami-0e75a596ad7454bb3"
+    ubuntu-amd64: ""
+    windows-amd64: ""
+    ubuntu-arm64: ""
```

Comment:
There seems to be an issue here.

Reply:
Yep, I would even say 2 issues:

- Updatecli lacks a way (or I don't know it) to fail when a source is empty
- The 0.29.0 tag for the packer VM templates failed to publish AWS images for these 3 AWS kinds

Reply:
The tag build on packer hasn't finished yet, that explains why these values are empty.